src/sentry/conf/server.py

```diff
@@ -1092,6 +1092,14 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
         ),
         "options": {"expires": 60 * 60 * 3},
     },
+    "schedule-daily-organization-reports": {
```

Comment:
Do we want to do a slow rollout for this task? I'm not sure how we could enable this for like 10% of orgs for instance - https://www.notion.so/Daily-and-Weekly-Issue-Summaries-a69786c9020543d986e349f89a3e78e6?d=66911497fa964ff69f86eca6c6896c49&pvs=4#ccafef79f6a14a9c960110560cda3992

Reply:
Yeah, I added a flag for this so we can do a slow rollout in Flagr.